articles/storage/blobs/lifecycle-management-policy-configure.md

```diff
@@ -228,6 +228,9 @@ A lifecycle management policy must be read or written in full. Partial updates a
 > [!NOTE]
 > The delete action of a lifecycle management policy won't work with any blob in an immutable container. With an immutable policy, objects can be created and read, but not modified or deleted. For more information, see [Store business-critical blob data with immutable storage](./immutable-storage-overview.md).
 
+> [!NOTE]
+> Kindly Note that while configuring the life cycle management policy if you add blob type as append blob, in that case for applying rule on base blob last accessed days option would not be available as it is not supported for append blob.
```

Comment:
On second glance, I believe what you are trying to call out is that if using the Azure portal to configure the policy, the "Last Accessed" option appears only if Block Blobs is selected. I suggest keeping the note inside of the Azure portal section of this procedure. In fact, we could just update the existing line "The Last accessed option is available only if you have enabled access time tracking." with "The Last accessed option is available only if you have enabled access time tracking and you've selected Block blobs as the blob type. With the term "Block blobs" bolded with markdown. Thoughts?

Reply:
The wording of this NOTE is confusing and doesn't follow MS style guidance. I'd edit it, but I'm confused by the intention of what it's trying to communicate. There was another PR to remove this, is that still the plan?